extensions/guacamole-auth-ldap/doc/examples/exampleConfigGroup.ldif

```diff
@@ -9,3 +9,4 @@ guacConfigParameter: port=5900
 guacConfigParameter: password=secret
 member: cn=user1,dc=example,dc=com
 member: cn=user2,dc=example,dc=com
+seeAlso: ou=admins,ou=groups,dc=example,dc=com
```

Comment:
Sorry, I know this issue is already closed and merged, but just wanted to note that:

A) I'm really happy this functionality is being added!
B) It took me a a while to get this working. I think this might be a typo in the documentation. For me to get the "seeAlso" to work, it had to point to the DN of an actual group not nested OU. 

Should the doc line be:
seeAlso: cn=admins,ou=groups,dc=example,dc=com
to avoid confusion?

Admittedly, AD is not my strong suit, so maybe I was doing something wrong at an OU level, but again, pointing seeAlso to a group DN did the trick for me.  

Thanks guys!

-JT